pkg/workflow/providers/multicluster/multicluster.go

```diff
@@ -159,6 +161,84 @@ func (p *provider) ListClusters(ctx wfContext.Context, v *value.Value, act wfTyp
 	return v.FillObject(clusters, "outputs", "clusters")
 }
 
+func (p *provider) ExpandTopology(ctx wfContext.Context, v *value.Value, act wfTypes.Action) error {
+	policiesRaw, err := v.LookupValue("inputs", "policies")
```

Comment:
let's use const for these string

Reply:
It is a style problem for all cue providers. If const should be used here, we might need to change all the cue providers to use the same style.

Reply:
yes, we can do that in the following PRs, and this is necessary